textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/queryrewriter/FuzzyTokenizer.java

```diff
@@ -16,12 +18,11 @@
  * For example -> ["newyorkcity"] -> ["new york city", "newyorkcity"]
  * Here, "new york city" is a returned phrase because "new", "york" and "city" are valid words in the word base.
  */
-public class FuzzyTokenizer
-{
+public class FuzzyTokenizer {
```

Comment:
I think the name `FuzzyTokenizer` is not very accurate.  Can we rename it to `QueryRewriter`?